### ISSUE TITLE
nick/buffer completion not working for /join or /query

### ISSUE THREAD

**Issue by jessebye** [wee-slack/wee-slack#380]
When I try to use `/join` or `/query` and tab-complete a nick/channel name, an error is logged:

```
python: error in function "nick_completion_cb"
python: stdout/stderr: Traceback (most recent call last):
python: stdout/stderr:   File "/Users/jbye/.weechat/python/autoload/wee_slack.py", line 761, in nick_completion_cb
python: stdout/stderr:     if current_channel is None or current_channel.members is None:
python: stdout/stderr: AttributeError: 'SlackTeam' object has no attribute 'members'
```

**Comment by trygveaa** [comment]
I don't get this error with current master, and completion of nicks is working. Completion of channels is not working though (but I don't think that's ever been implemented).

**Comment by drselump14** [comment]
I have same exact issue here ... suddenly it doesn't work at all and throw same message as above ..

**Comment by dromer** [comment]
I don't have any nick-completion.

Weechat 2.0.1

**Comment by selckin** [comment]
This happens in the "server" buffer (window 1), if you control x to the main slack window, so you are not in a channel, and then try to tab complete

```
> 133202         weechat     │ python: stdout/stderr: Traceback (most recent call last):
> 133202         weechat     │ python: stdout/stderr:   File "/home/selckin/.weechat/python/autoload/wee_slack.py", line 119, in wrapper
> 133202         weechat     │ python: stdout/stderr:     return f(*decode_from_utf8(args), **decode_from_utf8(kwargs))
> 133202         weechat     │ python: stdout/stderr:   File "/home/selckin/.weechat/python/autoload/wee_slack.py", line 826, in nick_completion_cb
> 133202         weechat     │ python: stdout/stderr:     if current_channel is None or current_channel.members is None:
> 133202         weechat     │ python: stdout/stderr: AttributeError: 'SlackTeam' object has no attribute 'members'
> 133202         weechat =!= │ python: error in function "nick_completion_cb"
```

**Comment by trygveaa** [comment]
Fixed with #638.
